pyproject.toml

```diff
@@ -0,0 +1,42 @@
+[tool.poetry]
+name = "fastlane-bot"
+version = "3.1.5"
+description = "Carbon Arbitrage Bot, an open-source arbitrage protocol, allows any user to perform arbitrage between Bancor ecosystem protocols and external exchanges and redirect arbitrage profits back to the protocol."
+authors = ["Bancor Network <platon@bancor.network>"]
+license = "MIT"
+readme = "README.md"
+packages = [{include = "fastlane_bot"}]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+psutil = "^5.9.6"
+packaging = "21.3"
+requests = "^2.31.0"
+python-dateutil = "^2.8.2"
+typing-extensions = "^4.7.1"
+python-dotenv = "^0.16.0"
+joblib = "^1.2.0"
+pandas = "^1.5.2"
+alchemy-sdk = "^0.1.1"
+pyarrow = "^11.0.0"
+networkx = "^3.0"
+cvxpy = "^1.3.1"
```

Comment:
can be removed

Reply:
done

Reply:
Tests are failing without `cvxpy = "^1.3.1"`, adding it back

Reply:
yeah. we also need to discuss this. those tests are for library code in the optimizer that is not in use in the bot proper.

Reply:
I can move it to dev dependencies then, if anything else should be moved in there let me know

Reply:
not sure that this is the ideal way of dealing with this -- maybe production users also want to run the test suite. Opening an issue.